package.json

```diff
@@ -10,19 +10,20 @@
   "description": "Tests for Azure REST API Specifications",
   "license": "MIT",
   "devDependencies": {
+    "@microsoft.azure/async-io": "^1.0.21",
+    "@microsoft.azure/literate": "^1.0.21",
+    "@microsoft.azure/polyfill": "^1.0.17",
+    "azure-storage": "^2.1.0",
```

Comment:
what do you need azure-storage dependency for?

Reply:
it was already here, just sorted in alphabetic order. not sure who has dependency on it, searching in the repo there is no reference, I can remove it

---
In reply to: [159792564](https://github.com/Azure/azure-rest-api-specs/pull/2211#discussion_r159792564) [](ancestors = 159792564)